runtime/testy/runtime.py

```diff
@@ -269,14 +264,14 @@ def runtimeTest(testNames):
   testNameRegex = re.compile(".*_setup")
   allTestNames = [testName[:-len("_setup")] for testName in dir(studentCode) if testNameRegex.match(testName)]
 
-  if len(testNames) == 0:
-    print("Running all tests")
-    testNames = allTestNames
-  else:
+  if testNames:
     for testName in testNames:
       if testName not in allTestNames:
         print("Error: {} not found.".format(testName))
         return
+  else:
```

Comment:
Hmm I was more following the principle of "always use positive `if` conditions with `if`-`else` statements, since they're easier to reason about".